packages/ckeditor5-mention/tests/mentionui.js

```diff
@@ -131,6 +131,41 @@ describe( 'MentionUI', () => {
 		} );
 	} );
 
+	describe( 'contextual balloon in rtl', () => {
```

Comment:
Let's move it to an existing "position" category below and have two dedicated tests:

```
describe( 'position', () => {
    ... // existing position category tests
    describe( 'UI language direction', () => {
        it( 'should set proper position list in RTL', () => {} );
        it( 'should set proper position list in LTR', () => {} );
```